<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GPIOBitSettingsWidget</class>
 <widget class="QWidget" name="GPIOBitSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>131</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelGPIOBitName">
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxDirection">
     <property name="title">
      <string>Direction</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBoxDirection">
        <item>
         <property name="text">
          <string>Input</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Output</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxSpeed">
     <property name="title">
      <string>Output speed</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBoxOutputSpeed">
        <item>
         <property name="text">
          <string>Low</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Medium</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>High</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Very high</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxInputPull">
     <property name="title">
      <string>Input pull</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBoxInputPull">
        <item>
         <property name="text">
          <string>No pull</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pull-up</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pull-down</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxOutputType">
     <property name="title">
      <string>Output type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBoxOutputType">
        <item>
         <property name="text">
          <string>Push-pull</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Open-drain</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Value</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBoxValue">
        <item>
         <property name="text">
          <string>Low</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>High</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
